scripts/TOUCHPOINTS.md

```diff
@@ -0,0 +1,33 @@
+*WHAT IS THIS?*
+This temporary script is meant as a stopgap to allow a person with a Touchpoint API Key to download results from a survey to their local device. The downloaded data can be used to for in-depth discovery of survey results.
+
+*HOW TO USE IT*
+In the _touchpoint.yml file, enter the form id you wish to retrieve results for as well as your API Key. Change the filename if you don't want to overwrite it.
+
+1. Modify _touchpoint.yml
+2. Run the script: ruby touchpoint.rb
+
+**_touchpoint.yml**
+form_id: The form you want results from
+api_key: Your API key
+start_date: Date starting of results you want
+end_date: Non-inclusive date
+filename: Target save name for csv file
+base_url: 'https://api.gsa.gov/analytics/touchpoints/v1/forms/'
+max_pages: Max number of pages to receive (# of calls to the API maximum)
+languages: The supported languages on the help site (en is assumed)
```

Comment:
this all ran together inline, I'd have done:
~~~
```
form_id: The form you want results from
...
languages: The supported languages on the help site (en is assumed)
```
~~~

Reply:
I put up #1170 to reformat this file